exec-unittests

```diff
@@ -161,43 +166,57 @@ if [[ -z "${subtst}" || "${subtst}" == "python" ]]; then
     fi
 fi
 
-_subtst_list="tool-scripts/datalog tool-scripts/postprocess tool-scripts util-scripts bench-scripts"
+_agent_subtst_list="tool-scripts/datalog tool-scripts/postprocess tool-scripts util-scripts bench-scripts"
```

Comment:
This seems like a weird place to define this.  I would think that this belongs at the top of the file, as a "configuration definition".  Failing that, I would place it near where it is used.